client/my-sites/domains/domain-management/components/domain/primary-flag.jsx

```diff
@@ -38,8 +31,9 @@ DomainPrimaryFlag.propTypes = {
 	translate: PropTypes.func.isRequired,
 };
 
-export default connect( state => {
-	return {
+export default flow(
```

Comment:
Why `flow` vs the old way? :) Curious.

Reply:
I like to refactor away any cases that I find of `functionThree( functionTwo( functionOne() ) )` type function nesting where possible because it's not a nicely 'scalable' pattern in that it can get very messy as more functions are wrapped.

It's not so much of an issue here since there are only 2 functions at play but I'd like to see it become more commonplace in Calypso to help folks avoid falling in to the nesting pattern.

There's some more discussion here: #10138

Reply:
Interesting! @Automattic/cobalt